onnxruntime/test/framework/function_test.cc

```diff
@@ -382,32 +385,25 @@ TEST(FunctionTest, AttrSaturate) {
 // TODO: change the expected value when this PR is merged in onnx:
 // https://github.com/onnx/onnx/pull/5246
 TEST(FunctionTest, AttrSaturateNan) {
+  Float8E4M3FNUZ f8(300.0);
+  float f8_r0 = f8.ToFloat();
+  std::cout << f8_r0 << std::endl;
```

Comment:
Remove the two lines when your test is done.

Reply:
code removed